src/Microsoft.Sbom.Api/Workflows/SBOMGenerationWorkflow.cs

```diff
@@ -98,22 +99,33 @@ public virtual async Task<bool> RunAsync()
                 {
                     sbomConfigs.ApplyToEachConfig(config => config.JsonSerializer.StartJsonObject());
 
-                    // Files section
-                    validErrors = await fileArrayGenerator.GenerateAsync();
+                    var manifestInfos = sbomConfigs.GetManifestInfos();
 
-                    // Packages section
-                    validErrors.Concat(await packageArrayGenerator.GenerateAsync());
-
-                    // External Document Reference section
-                    validErrors.Concat(await externalDocumentReferenceGenerator.GenerateAsync());
-
-                    // Relationships section
-                    validErrors.Concat(await relationshipsArrayGenerator.GenerateAsync());
+                    // If manifestInfos is empty (for example, this is the case for unit tests where GetManifestInfos() is not implemented), use the default SPDX 2.2 manifest info
```

Comment:
This feels like a test gap, but not one that needs to block the PR

Reply:
Will create a work item